flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -667,11 +668,9 @@ private void stopTaskExecutorServices() throws Exception {
 	}
 
 	@Override
-	public void failPartition(ExecutionAttemptID executionAttemptID) {
-		log.info("Discarding the results produced by task execution {}.", executionAttemptID);
```

Comment:
I am not sure whether to retain the previous log for tracing.

Reply:
I was thinking about it, the problem here is that previously it happened per task. Now, it gets a list of partitions without assumption that it is per task. JM will still log it in `sendReleaseIntermediateResultPartitionsRpcCall`. If more verbose mode is needed, debug level will enable per partition logging in `ResultPartitionManager.releasePartition` in TM.